### PR TITLE
feat: create userscript

### PR DESCRIPTION
**Pull request by maltoze**
we can use it in Tampermonkey or Greasemonkey now😁

image

image

### CODE REVIEW

**Comment by maltoze** [comment]
can use this script for testing.
```js
// ==UserScript==
// @name            openai-translator
// @namespace       https://github.com/yetone/openai-translator
// @version         0.1.0
// @description     基于 ChatGPT API 的划词翻译浏览器插件和跨平台桌面端应用
// @description:en  Browser extension and cross-platform desktop application for translation based on ChatGPT API
// @author          yetone,maltoze
// @icon64          https://cdn.jsdelivr.net/gh/yetone/openai-translator@main/public/icon.png
// @match           *://*/*
// @require         https://cdn.jsdelivr.net/gh/maltoze/openai-translator@userscript/dist/userscript/index.js
// @grant           GM.setValue
// @grant           GM.getValue
// @license         MIT
// ==/UserScript==

;(function () {})()
```

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,6 @@
 npm-debug.log
 node_modules/
-dist/
+dist/browser-extension/
```
Comment (yetone) [inline .gitignore:3]: Why not exclude all contents in the dist directory?

**Comment by yetone** [comment]
Why is it so cool!

File: .gitignore (hunk 1 above)
Comment (maltoze) [reply .gitignore:3]: We need to submit `dist/userscript/index.js` to github in order to generate a link from [jsdelivr](https://www.jsdelivr.com/github)

Comment (yetone) [reply .gitignore:3]: I see, maybe we should add `dist/electron` and `dist/tauri`, or put the contents of `userscript` into another directory.

Comment (maltoze) [reply .gitignore:3]: forgot that

**Comment by yetone** [comment]
I remember there was some code in this project that deletes the `dist` directory. Should we refactor it?

File: Makefile [hunk 2]
```diff
@@ -1,9 +1,11 @@
 VERSION ?= 0.1.0
 
+clean:
+	rm -rf ./dist/browser-extension/ ./dist/electron/ ./dist/tauri/
```
Comment (maltoze) [inline Makefile:4]: is this better? `find dist/* -not -path "dist/userscript*" -prune -exec rm -rf {} \;`

Comment (maltoze) [inline Makefile:4]:
```suggestion
	find dist/* -not -path "dist/userscript*" -prune -exec rm -rf {} \;
```

Comment (yetone) [reply Makefile:4]: Yes, I think this is better.

**Review by yetone — APPROVED** [review]

**Pull request merged**